src/NLog.Mongo/MongoTarget.cs

```diff
@@ -264,7 +343,9 @@ private BsonValue CreateException(Exception exception)
 #if !NETSTANDARD1_5
             if (exception is ExternalException external)
                 document.Add("ErrorCode", new BsonInt32(external.ErrorCode));
+            else if (exception.HResult != 0)
 #endif
+                document.Add("ErrorCode", new BsonInt32(exception.HResult));
```

Comment:
should this line be inside the #endif block?

Reply:
Removed breaking change. Just thought it would be nice.